kad-dht/README.md

```diff
@@ -268,23 +268,55 @@ heuristic of the value to make the decision.
 
 ### Content provider advertisement and discovery
 
-Nodes must keep track of which nodes advertise that they provide a given key
-(CID). These provider advertisements should expire, by default, after 24 hours.
-These records are managed through the `ADD_PROVIDER` and `GET_PROVIDERS`
+There are two things at play with regard to provider record (and therefore content)
+liveness and reachability:
+
+Content providers need to make sure that their content is reachable, despite peer churn
+and nodes that store and serve provider records need to make sure that the Multihashes whose 
+records they store are still served by the content provider.
+
+The following two parameters help cover both of these cases.
+1. **Provider Record Republish Interval (24hrs):** The content provider 
+needs to make sure that the nodes chosen to store the provider record 
+remain online when clients ask for the record. In order to 
+guarantee this, while taking into account the peer churn, content providers
+republish the records they want to provide every 24 hours.
+2. **Provider Record Expiration Interval (48hrs):** The network needs to provide
```

Comment:
Is there already a corresponding go-libp2p pull request to update these values?

Reply:
Sounds good. I suggest we only merge here once that pull request exists to make sure at least one implementation is up to date with the specification.